pkg/bbgo/interact.go

```diff
@@ -74,6 +56,15 @@ func (it *CoreInteraction) FilterStrategyByInterface(checkInterface interface{})
 	return strategies, found
```

Comment:
also, make FilterStrategyByInterface as pure simple function and you can pass `it.exchangeStrategies` as an input to it. 

make it a simple filter function, which can be easier for testing.

Reply:
done
is it because we can use FilterStrategyByInterface independently?